Add ControlPet game

mgschwan opened the pull request.

JelmerT commented on games/017_ControlPet/project.properties:
```diff
@@ -0,0 +1,3 @@
+dependencies.papertrail=0.0.5
+dependencies.hackerpet=0.1.0
```
Could this be updated to 0.2.2?

JelmerT commented:
Thanks for this PR, and your patience ;) on this @mgschwan !

Code is looking really good, and I want to merge it ASAP. I'll play around with it a little longer after that before I make more suggestions on improvements.

If you could just update to the latest version of the hackerpet lib, this is good to go for me!

mgschwan commented:
I moved to hackerpet library 0.2.2. It seems to work. The first time I tried it had a problem but since then it worked without problems. Could you please try if it works for you as well.

BTW: I renamed the game to 016_ControlPet because the other 016 game was removed
